.pre-commit-config.yaml

```diff
@@ -24,7 +24,6 @@ repos:
         args:
           - "--sg=build/**"  # Matches "${ISORT_YAPF_EXCLUDES[@]}"
           - "--sg=sky/skylet/providers/ibm/**"
```

Comment:
The `format.sh` script limits files to this range, but .github/workflows/format.yml does not. Remove the limitation in format.sh to ensure consistency with the final result (format.yml).